modules/drivers/mongo/src/metabase/driver/mongo.clj

```diff
@@ -415,6 +415,7 @@
                               :standard-deviation-aggregations true
                               :test/jvm-timezone-setting       false
                               :identifiers-with-spaces         true
+                              :saved-question-sandboxing       false
```

Comment:
Are there tests exercising mongo sandboxing specifically? I would like to see real queries being run and tested.